webapp/channels/src/components/widgets/inputs/dropdown_input_hybrid.tsx

```diff
@@ -57,23 +57,23 @@ const baseStyles = {
         ...provided,
         zIndex: 99999999,
     }),
-} satisfies StylesConfig<OptionType, boolean>;
+} satisfies StylesConfig<OptionType, false>;
 
-const IndicatorsContainer = <T extends OptionType>(props: IndicatorsContainerProps<T>) => (
+const IndicatorsContainer = <T extends OptionType>(props: IndicatorsContainerProps<T, false>) => (
```

Comment:
These changes were necessary because some changes caused stricter type checking and onValueChange only supports a `SingleValue` object. 

`const onValueChange = (event: SingleValue<T>) => {`